YSI_Coding/y_malloc/y_malloc_funcs.inc

```diff
@@ -340,6 +371,54 @@ P:D(Malloc_NextSlot(slot));
 
 #endif
 
+/*-------------------------------------------------------------------------*//**
+ * <param name="array">Data unit to put information in.</param>
+ * <param name="index">Index in the unit.</param>
+ * <param name="value">Float value to insert</param>
+ * <remarks>
+ *  Displays errors in secure mode.
+ * </remarks>
+ *//*------------------------------------------------------------------------**/
+
+#if defined YSI_MALLOC_SECURE
+
+	Float:Malloc_GetFloat(Alloc:array, index) {
+		P:3("Malloc_GetFloat: array = %d, index = %d.", _:array, index);
+		return Malloc_Get(array, index);
```

Comment:
You don't override the return tag here.

Reply:
may I know why? I guess return 1; would do?